<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" itemscope="" itemtype="http://schema.org/WebPage">

<!-- Mirrored from sites.google.com/site/drrabhishekkumar/tools-and-softwares/regulatory-genomic-tools by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 25 Feb 2019 18:04:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="chrome=1" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { (function(){function e(a){this.t={};this.tick=function(a,c,b){this.t[a]=[void 0!=b?b:(new Date).getTime(),c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(h){}};this.tick("start",null,a)}var a;if(window.performance)var d=(a=window.performance.timing)&&a.responseStart;var f=0<d?new e(d):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_","_wtsrt",
d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})(); })()
</script>
<link rel="shortcut icon" type="image/x-icon" href="http://www.google.com/images/icons/product/sites-16.ico" />
<link rel="apple-touch-icon" href="../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/images/apple-touch-icon.png" type="image/png" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { function d(a){return document.getElementById(a)}window.byId=d;function g(a){return a.replace(/^\s+|\s+$/g,"")}window.trim=g;var h=[],k=0;window.JOT_addListener=function(a,b,c){var f=new String(k++);a={eventName:a,handler:b,compId:c,key:f};h.push(a);return f};window.JOT_removeListenerByKey=function(a){for(var b=0;b<h.length;b++)if(h[b].key==a){h.splice(b,1);break}};window.JOT_removeAllListenersForName=function(a){for(var b=0;b<h.length;b++)h[b].eventName==a&&h.splice(b,1)};
window.JOT_postEvent=function(a,b,c){var f={eventName:a,eventSrc:b||{},payload:c||{}};if(window.JOT_fullyLoaded)for(b=h.length,c=0;c<b&&c<h.length;c++){var e=h[c];e&&e.eventName==a&&(f.listenerCompId=e.compId||"",(e="function"==typeof e.handler?e.handler:window[e.handler])&&e(f))}else window.JOT_delayedEvents.push({eventName:a,eventSrc:b,payload:c})};window.JOT_delayedEvents=[];window.JOT_fullyLoaded=!1;
window.JOT_formatRelativeToNow=function(a,b){a=((new Date).getTime()-a)/6E4;if(1440<=a||0>a)return null;var c=0;60<=a&&(a/=60,c=2);2<=a&&c++;return b?window.JOT_siteRelTimeStrs[c].replace("__duration__",Math.floor(a)):window.JOT_userRelTimeStrs[c].replace("__duration__",Math.floor(a))}; })()
</script>
<script>

  

  var breadcrumbs = [{"path":"/site/drrabhishekkumar/tools-and-softwares","deleted":false,"title":"Tools and Softwares","dir":"ltr"},{"path":"/site/drrabhishekkumar/tools-and-softwares/regulatory-genomic-tools","deleted":false,"title":"Regulatory Genomic Tools","dir":"ltr"}];
  var JOT_clearDotPath = '../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/images/cleardot.gif';

  
  var JOT_userRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

  
  

  

  var webspace = {"gvizGstaticVersion":"current","enableAnalytics":false,"pageSharingId":"jotspot_page","codeembeds":{"outerIframeSrc":"https://www.gstatic.com/jotspot/embeds/code/0f08d42392f2000e7e3f3daf5b427a43/outer_iframe.html","innerIframeSrc":"https://1585240993-jotspot-embeds.googleusercontent.com/code/8d87fa64604b2a11fae2ed06104c58d3/inner_iframe.html"},"enableUniversalAnalytics":false,"sharingPolicy":"OPENED","siteTitle":"Dr. Abhishek Kumar","jot2atari":{"eligibility":"ineligible"},"onepickUrl":"https://docs.google.com/picker","adsensePublisherId":null,"features":{"oAuthForChartsApi":true,"moreMobileStyleImprovements":null,"googleChartsOverGstatic":false,"picasaAlbumInsert":false,"removeGooglePlusWidgets":true,"enableJot2Atari":true,"plusBadge":false},"isPublic":true,"newSitesBaseUrl":"https://sites.google.com","isConsumer":true,"serverFlags":{"jot2AtariLearnMoreUrl":"https://support.google.com/sites/answer/7035197"},"domainAnalyticsAccountId":"","plusPageId":"","signInUrl":"https://accounts.google.com/AccountChooser?continue\u003dhttps://sites.google.com/site/drrabhishekkumar/tools-and-softwares/regulatory-genomic-tools\u0026service\u003djotspot","analyticsAccountId":"","scottyUrl":"/_/upload","homePath":"/","siteNoticeUrlEnabled":null,"plusPageUrl":"","adsensePromoClickedOrSiteIneligible":true,"csiReportUri":"https://gg.google.com/csi","sharingId":"jotspot","termsUrl":"//www.google.com/intl/en/policies/terms/","gvizVersion":1,"editorResources":{"sitelayout":["https://ssl.gstatic.com/sites/p/6b8a44/system/app/css/sitelayouteditor.css"],"text":["https://ssl.gstatic.com/sites/p/6b8a44/system/js/codemirror.js","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/codemirror_css.css","../../../../ssl.gstatic.com/sites/p/6b8a44/system/js/trog_edit__en.js","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/trogedit.css","../_/rsrc/1550133439000/system/app/css/editor.css","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/codeeditor.css","/site/drrabhishekkumar/_/rsrc/1550133439000/system/app/css/camelot/editor-jfk.css"]},"sharingUrlPrefix":"/_/sharing","isAdsenseEnabled":true,"domain":"defaultdomain","baseUri":"/site/drrabhishekkumar","name":"drrabhishekkumar","siteTemplateId":false,"siteNoticeRevision":null,"siteNoticeUrlAddress":null,"siteNoticeMessage":null,"page":{"isRtlLocale":false,"canDeleteWebspace":null,"isPageDraft":null,"parentPath":"/tools-and-softwares","parentWuid":"wuid:gx:326206f311529336","siteLocale":"en","timeZone":"America/Los_Angeles","type":"text","title":"Regulatory Genomic Tools","locale":"en","wuid":"wuid:gx:767b5c2870871d3d","revision":3,"path":"/tools-and-softwares/regulatory-genomic-tools","isSiteRtlLocale":false,"pageInheritsPermissions":null,"name":"regulatory-genomic-tools","canChangePath":true,"state":"","properties":{},"bidiEnabled":false,"currentTemplate":{"path":"/system/app/pagetemplates/text","title":"Web Page"}},"canPublishScriptToAnyone":true,"user":{"keyboardShortcuts":true,"sessionIndex":"","onePickToken":"","guest_":true,"displayNameOrEmail":"guest","userName":"guest","uid":"","renderMobile":false,"domain":"","namespace":"","hasWriteAccess":false,"namespaceUser":false,"primaryEmail":"guest","hasAdminAccess":false},"gadgets":{"baseUri":"/site/drrabhishekkumar/system/app/pages/gadgets"}};
  webspace.page.breadcrumbs = breadcrumbs;

  
  var JOT_siteRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

</script>
<script type="text/javascript">
                window.jstiming.load.tick('scl');
              </script>
<meta name="title" content="Regulatory Genomic Tools - Dr. Abhishek Kumar" />
<meta itemprop="name" content="Regulatory Genomic Tools - Dr. Abhishek Kumar" />
<meta property="og:title" content="Regulatory Genomic Tools - Dr. Abhishek Kumar" />
<style type="text/css">
</style>
<link rel="stylesheet" type="text/css" href="../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/themes/microsport/standard-css-microsport-ltr-ltr.css" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1550133439000/system/app/css/overlay4194.css?cb=microsport150goog-ws-nav-left-noheadnone30themedefaultstandard" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1550133439000/system/app/css/camelot/allthemes-view.css" />
<!--[if IE]>
          <link rel="stylesheet" type="text/css" href="/site/drrabhishekkumar/system/app/css/camelot/allthemes%2die.css" />
        <![endif]-->
<title>Regulatory Genomic Tools - Dr. Abhishek Kumar</title>
<script type="text/javascript">
                window.jstiming.load.tick('cl');
              </script>
</head>
<body xmlns="http://www.google.com/ns/jotspot" id="body" class=" en            ">
<div id="sites-page-toolbar" class="sites-header-divider">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-status" class="sites-status" style="display:none;"><div id="sites-notice" class="sites-notice" role="status" aria-live="assertive"> </div></div>
</div>
<div id="sites-chrome-everything-scrollbar">
<div id="sites-chrome-everything" class="">
<div id="sites-chrome-page-wrapper" style="direction: ltr">
<div id="sites-chrome-page-wrapper-inside">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-chrome-header-wrapper" style="">
<table id="sites-chrome-header" class="sites-layout-hbox" cellspacing="0" style="">
<tr class="sites-header-primary-row" id="sites-chrome-userheader">
</tr>
<tr class="sites-header-secondary-row" id="sites-chrome-horizontal-nav">
<td colspan="2" id="sites-chrome-header-horizontal-nav-container" role="navigation">
<div class="sites-header-nav"><ul class="sites-header-nav-container-boxes"><li class="unselected"><a class="sites-navigation-link unselected" href="../home.html">Home</a></li></ul><div style="clear: both;"></div></div>
</td>
</tr>
</table>
</div>
<div id="sites-chrome-main-wrapper">
<div id="sites-chrome-main-wrapper-inside">
<table id="sites-chrome-main" class="sites-layout-hbox" cellspacing="0" cellpadding="{scmCellpadding}" border="0">
<tr>
<td id="sites-chrome-sidebar-left" class="sites-layout-sidebar-left initial" style="width:150px">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_2bd" class="sites-embed" role="navigation"><div class="sites-embed-content sites-sidebar-nav"><ul role="navigation" jotId="navList" class="has-expander"><li class="topLevel nav-first parent " wuid="gx:7181a2541346e1b0"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../home.html" jotId="wuid:gx:7181a2541346e1b0" class="sites-navigation-link topLevel">Home</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/awards-fellowships.html" jotId="wuid:gx:7b30df081f9ba5b9" class="sites-navigation-link">Awards &amp; Fellowships</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/cv.html" jotId="wuid:gx:739bc597c14732cb" class="sites-navigation-link">CV</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/education.html" jotId="wuid:gx:69568c90e4cd4346" class="sites-navigation-link">Education</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/lebenslauf.html" jotId="wuid:gx:9fbe71b7c99d1d8" class="sites-navigation-link">Lebenslauf</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/profile.html" jotId="wuid:gx:2db26263ae4fafb8" class="sites-navigation-link">Profile</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../bgiseq.html" jotId="wuid:gx:7c73f89c99b34613" class="sites-navigation-link topLevel">BGISEQ</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../collaborators.html" jotId="wuid:gx:daa1485044a82dd" class="sites-navigation-link topLevel">Collaborators</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../contact-me.html" jotId="wuid:gx:43b763280cdae9b1" class="sites-navigation-link topLevel">Contact Me</a></div></li><li class="topLevel parent " wuid="gx:9596e0d0a4c3aec"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../developments.html" jotId="wuid:gx:9596e0d0a4c3aec" class="sites-navigation-link topLevel">Developments</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/comparative-genomics-pipeline.html" jotId="wuid:gx:70d8b161a82ed9c0" class="sites-navigation-link">Comparative Genomics pipeline</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/pipeline-genetic-variants-and-phylogeny.html" jotId="wuid:gx:2c92bc6f3605332d" class="sites-navigation-link">Genetic variants and phylogeny</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/marine-fungal-genomics-pipeline.html" jotId="wuid:gx:d2a98c54427d68c" class="sites-navigation-link">Marine fungal genomics pipeline</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/proteogenomics-pipelines.html" jotId="wuid:gx:61bdfb4e8d513b78" class="sites-navigation-link">Proteogenomics pipelines</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/insect-genomics-transcriptomics.html" jotId="wuid:gx:4cfd8657799fdfac" class="sites-navigation-link">Transcriptomic analyses pipeline</a></div></li></ul></li><li class="topLevel parent " wuid="gx:19dd9f1e0b68b591"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../hallmarks-of-genomics-bioinformatics.html" jotId="wuid:gx:19dd9f1e0b68b591" class="sites-navigation-link topLevel">Hallmarks of Genomics &amp; Bioinformatics</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../hallmarks-of-genomics-bioinformatics/nanopore.html" jotId="wuid:gx:1b0d3f710d95a120" class="sites-navigation-link">Nanopore</a></div></li></ul></li><li class="topLevel parent " wuid="gx:43d574afaf9eb7a1"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../job-search.html" jotId="wuid:gx:43d574afaf9eb7a1" class="sites-navigation-link topLevel">Job Search</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../job-search/jobs-in-germany.html" jotId="wuid:gx:1e8cd417bd04e76b" class="sites-navigation-link">Jobs In Germany</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../list-of.html" jotId="wuid:gx:465464df5caf5847" class="sites-navigation-link topLevel">List of Genomes</a></div></li><li class="topLevel parent " wuid="gx:9046d8ca199764a"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../living-in-germany.html" jotId="wuid:gx:9046d8ca199764a" class="sites-navigation-link topLevel">Living in Germany</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../living-in-germany/disability-and-life-in-germany.html" jotId="wuid:gx:3ed11e3d77709aac" class="sites-navigation-link">Disability and life in Germany</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../management-portfolio.html" jotId="wuid:gx:111bc9a268c7b7b8" class="sites-navigation-link topLevel">Management Portfolio</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../press-and-media.html" jotId="wuid:gx:5edb79290ac33e7b" class="sites-navigation-link topLevel">Press and Media</a></div></li><li class="topLevel parent " wuid="gx:6dd320bd7201d91c"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../professional-roles.html" jotId="wuid:gx:6dd320bd7201d91c" class="sites-navigation-link topLevel">Professional Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/editorial-board-member.html" jotId="wuid:gx:3ec0c9dfea1c55c8" class="sites-navigation-link">Editorial Board member</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/reviewer.html" jotId="wuid:gx:5ab4a629b1e809fa" class="sites-navigation-link">Reviewer</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/scientific-society-membership.html" jotId="wuid:gx:fc379cf6763ccee" class="sites-navigation-link">Scientific Society Membership</a></div></li></ul></li><li class="topLevel parent " wuid="gx:334fd11dea82d559"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../publications.html" jotId="wuid:gx:334fd11dea82d559" class="sites-navigation-link topLevel">Publication Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/books.html" jotId="wuid:gx:11b64f0213fc80c6" class="sites-navigation-link">Books</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/conference-posters.html" jotId="wuid:gx:512c5367927fe6f2" class="sites-navigation-link">Conference Posters</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/corresponding-author.html" jotId="wuid:gx:593aadc27dbbef19" class="sites-navigation-link">Corresponding Author</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/first-author.html" jotId="wuid:gx:64008ae6efa15d9d" class="sites-navigation-link">First Author</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/if-wise.html" jotId="wuid:gx:41d920a865179998" class="sites-navigation-link">IF-wise</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/list-of-posters-presented.html" jotId="wuid:gx:2dbbb3814063aadc" class="sites-navigation-link">List of posters presented</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/talks.html" jotId="wuid:gx:6bb48c5996447827" class="sites-navigation-link">Talks</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/top-publications.html" jotId="wuid:gx:5aebbe9a44453899" class="sites-navigation-link">Top Publications</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/top-listed.html" jotId="wuid:gx:65ef0b68b78e8942" class="sites-navigation-link">Top-Listed</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/year-wise-publication-list.html" jotId="wuid:gx:6df2004a1dd59cb3" class="sites-navigation-link">Year wise publication list</a></div></li></ul></li><li class="topLevel parent " wuid="gx:28368d211dc65250"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../research-profile.html" jotId="wuid:gx:28368d211dc65250" class="sites-navigation-link topLevel">Research Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../research-profile/junior-research-fellow-ncbs.html" jotId="wuid:gx:517c1f5fe5dd02b0" class="sites-navigation-link">Junior Research Fellow (NCBS)</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../research-profile/msc-biotechnology.html" jotId="wuid:gx:26216082bbe83fd9" class="sites-navigation-link">MSc (Biotechnology)</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../research-profile/phd-bioinformatics.html" jotId="wuid:gx:1115a41e1bbcca84" class="sites-navigation-link">PhD (Bioinformatics)</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../research-profile/research-experiences.html" jotId="wuid:gx:1b31dde1ae53a3a2" class="sites-navigation-link">Research Experiences</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../research-profile/research-interests.html" jotId="wuid:gx:34d09da5e6febfea" class="sites-navigation-link">Research interests</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../synteny-tools.html" jotId="wuid:gx:22f18bc3a7b2eef2" class="sites-navigation-link topLevel">Synteny tools</a></div></li><li class="topLevel parent " wuid="gx:30c77ffaa0a2a074"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../teaching.html" jotId="wuid:gx:30c77ffaa0a2a074" class="sites-navigation-link topLevel">Teaching Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../teaching/python-programming.html" jotId="wuid:gx:220765ba8541ca5d" class="sites-navigation-link">Python Programming</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../teaching/training-on-reference-management-tools.html" jotId="wuid:gx:2c8565b3fd60619c" class="sites-navigation-link">Training on Reference Management Tools</a></div></li></ul></li><li class="topLevel parent " wuid="gx:326206f311529336"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../tools-and-softwares.html" jotId="wuid:gx:326206f311529336" class="sites-navigation-link topLevel">Tools and Softwares</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="all-sanger-tools.html" jotId="wuid:gx:689418979614cc74" class="sites-navigation-link">All Sanger tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="genetic-variant-analysis-tools.html" jotId="wuid:gx:439f968597b6f503" class="sites-navigation-link">Genetic Variant Analysis tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="genomics-tools.html" jotId="wuid:gx:5361d3e140dd53a9" class="sites-navigation-link">Genomics tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="missense-variants-ranking.html" jotId="wuid:gx:7316b4ab8208c56a" class="sites-navigation-link">Missense Variants Ranking</a></div></li><li class=""><div class="current-bg" jotId="wuid:gx:767b5c2870871d3d" dir="ltr" style="padding-left: 38px;">Regulatory Genomic Tools</div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="single-cell-genomics-transcriptomics.html" jotId="wuid:gx:7acb3cef3e39df30" class="sites-navigation-link">Single Cell Genomics &amp; Transcriptomics</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="structural-modelling-of-proteins.html" jotId="wuid:gx:1d9de95549bc3d48" class="sites-navigation-link">Structural modelling of proteins</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="tools-for-database-id-conversion.html" jotId="wuid:gx:7a739490df4e1b34" class="sites-navigation-link">tools for database id conversion</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="tools-for-pathway-analyses.html" jotId="wuid:gx:36848779604c56c7" class="sites-navigation-link">Tools for Pathway analyses</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../useful-genomics-bioinformatics-links.html" jotId="wuid:gx:4f1fb032abe3227a" class="sites-navigation-link topLevel">Useful Genomics &amp; Bioinformatics links</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../variant2structure.html" jotId="wuid:gx:36befd3180c1797c" class="sites-navigation-link topLevel">Variant2Structure</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../videos.html" jotId="wuid:gx:8fe2237e3b0dd10" class="sites-navigation-link topLevel">Videos</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../system/app/pages/sitemap/hierarchy.html" jotId="wuid:gx:7700fb28d3d3353c" class="sites-navigation-link topLevel">Sitemap</a></div></li></ul></div></div>
</td>
<td id="sites-canvas-wrapper">
<div id="sites-canvas" role="main">
<div id="goog-ws-editor-toolbar-container"> </div>
<div xmlns="http://www.w3.org/1999/xhtml" id="title-crumbs" style="">
<A href="../tools-and-softwares.html" dir="ltr">Tools and Softwares</A>‎ &gt; ‎
  </div>
<h3 xmlns="http://www.w3.org/1999/xhtml" id="sites-page-title-header" style="" align="left">
<span id="sites-page-title" dir="ltr" tabindex="-1" style="outline: none">Regulatory Genomic Tools</span>
</h3>
<div id="sites-canvas-main" class="sites-canvas-main">
<div id="sites-canvas-main-content">
<table xmlns="http://www.w3.org/1999/xhtml" cellspacing="0" class="sites-layout-name-one-column sites-layout-hbox"><tbody><tr><td class="sites-layout-tile sites-tile-name-content-1"><div dir="ltr"><br /><div><br /></div><div><table style="border-collapse:collapse;border:1px solid black;padding:10px;color:rgb(77,77,77);font-family:Roboto,sans-serif;font-size:16px;width:939px"><tbody><tr bgcolor="#f2f2f2"><th style="border:1px solid black;padding:10px;font-size:18px" width="20%">Software</th><th style="border:1px solid black;padding:10px;font-size:18px" width="10%">Data Type</th><th style="border:1px solid black;padding:10px;font-size:18px" width="70%">Description</th></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://promoter.bx.psu.edu/hi-c/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">3D Genome Browser</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C<br />Virtual 4C</td><td style="border:1px solid black;padding:10px;font-size:18px">The 3D Genome Browser provides heatmap virtualizations of the Hi-C intra-chromosomal contract matrices. The browser allows the exploration of potential cis-regulatory elements and/or topologically associating domains (TADs) for the gene or genomic feature of interest.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://github.com/rr1859/R.4Cker" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">4C-ker</a></td><td style="border:1px solid black;padding:10px;font-size:18px">4C</td><td style="border:1px solid black;padding:10px;font-size:18px">4C-ker is a method developed to analyze 4C-Seq (circularized chromosome conformation capture) data. 4C-ker is a Hidden-Markov Model based pipeline that identifies regions throughout the genome that interact with the 4C bait locus.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://ibi.hzau.edu.cn/3dmodel/index.php" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">AutoChrom3D</a></td><td style="border:1px solid black;padding:10px;font-size:18px">3C, 4C, 5C<br />Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">AutoChrom3D is an automatic pipeline to model chromatin regions with genomic size ranging from hundreds of kilobases to megabases at 8 kb resolution.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://projects.cbio.mines-paristech.fr/centurion/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Centurion</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A method that jointly calls all centromeres in a genome-wide Hi-C contact map.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://chiapet.gis.a-star.edu.sg/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">ChIA-PET Browser</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">A software package for automatic processing of ChIA-PET sequence data, including linker filtering, mapping tags to reference genomes, identifying protein binding sites and chromatin interactions, and displaying the results on a graphical genome browser.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://folk.uio.no/jonaspau/chiasig/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">ChiaSig</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">ChiaSig finds significant interactions based on the NCHG model.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://regulatorygenomicsgroup.org/chicago" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">CHiCAGO</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A set of tools for calling significant interactions in Capture Hi-C data, such as Promoter Capture Hi-C.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://bioinfo.sls.kyushu-u.ac.jp/chromcontact/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">ChromContact</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">ChromContact is a web server for analyzing spatial contact of chromosomes from the publicly available Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://cran.r-project.org/web/packages/chromoR/index.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">chromoR</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">chromoR provides users with a statistical pipeline for analysing chromosomal interactions data (Hi-C data).</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://biogpu.ddns.comp.nus.edu.sg/~chipseq/ChromSDE/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">ChromSDE</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A deterministic method which applies semi-definite programming techniques to find the best structure fitting the observed data and uses golden section search to find the correct parameter for converting the contact frequency to spatial distance.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://apps.cytoscape.org/apps/cytohicplugin" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">CytoHiC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A Cytoscape plugin, which allow users to view and compare spatial maps of genomic landmarks, based on normalized Hi-C datasets.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.bioconductor.org/packages/release/bioc/html/diffHic.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">diffHiC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A software package for the detection of differential interactions from Hi-C data. diffHic provides methods for read pair alignment and processing, counting into bin pairs, filtering out low-abundance events and normalization of trended or CNV-driven biases.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://cran.r-project.org/web/packages/FisHiCal/index.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">FisHiCal</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">FisHiCal integrates Hi-C and FISH data, offering a modular and easy-to-use tool for chromosomal spatial analysis.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://noble.gs.washington.edu/proj/fit-hi-c/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Fit-Hi-C</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">Fit-Hi-C is a tool for assigning statistical confidence estimates to intra-chromosomal contact maps produced by genome-wide genome architecture assays such as Hi-C.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://bioconductor.org/packages/release/bioc/html/GenomicInteractions.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">GenomicInteractions</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C<br />ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">R package for handling Genomic interaction data, such as ChIA-PET/Hi-C, annotating genomic features with interaction information and producing various plots/statistics.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.genomegitar.org/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">GITAR</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">GITAR (Genome Interaction Tools and Resources) is a standardized way to analyze and visualize genomic interaction data. The tool is composed of two modules: HiCtool (a standardized Python library for processing and visualizing Hi-C data) and processed data (a collection of datasets processed using HiCtool). The pipeline performs also TADs analysis.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.givengine.org/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">GIVE</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C<br />ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">GIVE is an open source programming library that allows anyone with HTML programming experience to build custom genome browser websites or apps. With a few lines of codes, one can add to a personal webpage an interactive genome browser that host custom data. It typically takes less than half a day to build a genome browser website with GIVE.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">GOTHiC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A Hi-C analysis package using a cumulative binomial test to detect interactions between distal genomic loci that have significantly more reads than expected by chance in Hi-C experiments.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://hyperbrowser.uio.no/3d/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiBrowse</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A statistical web toolkit for analyzing, interpreting and visualizing genome-wide chromosome conformation capture data, such as Hi-C, TCC, GCC and similar.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0678-x" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiCdat</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">HiCdat provides a simple graphical user interface for data pre-processing and a collection of higher-level data analysis tools implemented in R.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://hic.umassmed.edu/welcome/welcome.php" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Hi-C Data Browser</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">The tool enables the generation of genome-wide 3D proximity maps. You will also be able to visually navigate the dataset and explore Hi-C proximity maps of entire human genomes.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://github.com/HiC-inspector/HiC-inspector" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiC-inspector</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A bioinformatics pipeline for the automated analysis of data generated by high-throughput chromatin conformation capture (HiC).</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.people.fas.harvard.edu/~junliu/HiCNorm/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiCNorm</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A parametric model to remove systematic biases in the raw Hi-C contact maps, resulting in a simple, fast, yet accurate normalization procedure.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://zhoulab.usc.edu/Hi-Corrector/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Hi-Corrector</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A fast, scalable and memory-efficient package for normalizing large-scale Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://compgenomics.weizmann.ac.il/tanay/?page_id=283" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Hi-C Pipeline</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">Hicpipe is a set of scripts and programs that correct Hi-C contact maps, given a list restriction enzyme sites and mapped paired reads.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://github.com/kcakdemir/HiCPlotter" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiCPlotter</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">An easy-to-use open-source visualization tool to facilitate juxtaposition of Hi-C matrices with diverse genomic assay outputs, as well as to compare interaction matrices between various conditions.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0831-x" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiC-Pro</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">HiC-Pro is an optimized and flexible pipeline for processing Hi-C data from raw reads to normalized contact maps.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://cran.r-project.org/web/packages/HiCseg/index.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiCseg</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">This package allows you to detect domains in HiC data by rephrasing this problem as a two-dimensional segmentation issue.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://sourceforge.net/projects/hictools/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">hiCtools</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">HiC is high-throughput sequencing technology applied to chromatin conformation techniques. This collection of tools stream-lines the processing of HiC data from raw sequence to contact matrices and beyond.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.bioinformatics.babraham.ac.uk/projects/hicup/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiCUP</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A tool for mapping and performing quality control on Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0806-y" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiFive</a></td><td style="border:1px solid black;padding:10px;font-size:18px">5C<br />Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">The HiFive tool suite provides efficient data handling and a variety of normalization approaches for easy, fast analysis and method comparison.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://higlass.gehlenborglab.org/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiGlass</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Interaction matrices</td><td style="border:1px solid black;padding:10px;font-size:18px">HiGlass is a tool for displaying and comparing large matrices within a web page. Matrices are typically visualized as heatmaps. In the case of large matrices, the heatmaps are too large to render all at once. Instead, we aggregate their values and display summaries at lower resolution while allowing one to zoom in and explore them in greater detail.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://wanglab.pcbi.upenn.edu/hippie/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HIPPIE</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">HIPPIE is the workflow for analyzing batches of Hi-C paired-end reads in compressed FASTQ format (.fastq.gz) and predict enhancer-target gene interactions.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.bioconductor.org/packages//2.10/bioc/html/HiTC.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HiTC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">5C<br />Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">The package explores high-throughput 'C' data such as 5C or Hi-C.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.unc.edu/~yunmli/HMRFBayesHiC/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HMRFBayesHiC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A hidden Markov random field based Bayesian peak caller to identify long range chromatin interactions from Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://homer.ucsd.edu/homer/interactions/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HOMER</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">HOMER contains several programs and analysis routines to facilitate the analysis of Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://github.com/huangjialiangcn/HubPredictor" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">HubPredictor</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A computational model integrating Hi-C and histone mark ChIP-seq data to predict two important features of chromatin organization: chromatin interaction hubs and topologically associated domain (TAD) boundaries.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://mirnylab.bitbucket.io/hiclib/index.html?" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">ICE</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A computational pipeline that integrates a strategy to map sequencing reads with a data-driven method for iterative correction of biases, yielding genome-wide maps of relative contact probabilities.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.aidenlab.org/juicebox/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Juicebox</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">Software for visualizing data from Hi-C and other proximity mapping experiments.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://aidenlab.org/juicer/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Juicer</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">Juicer is a one-click system for analyzing loop-resolution Hi-C experiments. Specifically, the software is an end-to-end automated pipeline for converting raw reads into Hi-C maps and loop calls using only a single command.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://shendurelab.github.io/LACHESIS/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">LACHESIS</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A software tool to measure the thread of life. LACHESIS exploits contact probability map data (e.g. from Hi-C) for chromosome-scale de novo genome assembly.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="https://github.com/dphansti/mango" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">Mango</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">A complete ChIA-PET data analysis pipeline that provides statistical confidence estimates for interactions and corrects for major sources of bias including differential peak enrichment and genomic proximity.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://www.stat.osu.edu/~statgen/SOFTWARE/MDM/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">MDM</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">MDM provides functions for the analysis of chromatin interactions using MC_DIST model, Two-Step model and One-Step model, based on ChIA-PET data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://bioinfo.au.tsinghua.edu.cn/member/xwwang/MICCusage/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">MICC</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">An easy-to-use R package to detect chromatin interactions from ChIA-PET sequencing data. By applying a Bayesian mixture model to systematically remove random ligation and random collision noise, MICC could identify chromatin interactions with a significantly higher sensitivity than existing methods at the same false discovery rate.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://calla.rnet.missouri.edu/mogen/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">MOGEN</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">A 3D chromosome reconstruction method to make it capable of reconstructing 3D models of genomes from both intra- and inter-chromosomal Hi-C contact data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0075146" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">NuChart</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">An R Package to Study Gene Spatial Neighbourhoods with Multi-Omics Annotations.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://cbio.ensmp.fr/~nvaroquaux/pastis/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">PASTIS</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">Novel approach to infer a consensus three-dimensional structure of a genome from Hi-C data. The method incorporates a statistical model of the contact counts, assuming that the counts between two loci follow a Poisson distribution whose intensity decreases with the physical distances between the loci.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://bioconductor.org/packages/release/bioc/html/R3CPET.html" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">R3CPET</a></td><td style="border:1px solid black;padding:10px;font-size:18px">ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">The package provides a method to infer the set of proteins that are more probably to work together to maintain chormatin interaction given a ChIA-PET experiment results.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://compbio.cs.brown.edu/projects/tadtree/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">TADtree</a></td><td style="border:1px solid black;padding:10px;font-size:18px">Hi-C</td><td style="border:1px solid black;padding:10px;font-size:18px">An algorithm for the identification of hierarchical topological domains in Hi-C data.</td></tr><tr><td style="border:1px solid black;padding:10px;font-size:18px"><a href="http://epigenomegateway.wustl.edu/" style="color:rgb(43,43,43)" target="_blank" rel="nofollow">WashU Epigenome Browser</a></td><td style="border:1px solid black;padding:10px;font-size:18px">5C<br />Hi-C<br />ChIA-PET</td><td style="border:1px solid black;padding:10px;font-size:18px">The browser supports multiple types of long-range genome interaction data. This enables investigators to explore epigenomic data in the context of higher-order chromosomal domains and to generate multiple types of intuitive, publication-quality figures of interactions.</td></tr></tbody></table><br /></div><div>Source: https://www.4dnucleome.org/software.html</div></div></td></tr></tbody></table>
</div> 
</div> 
<div id="sites-canvas-bottom-panel">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-subpages"> </div>
<div id="sites-attachments-container">
</div>
<a xmlns="http://www.w3.org/1999/xhtml" name="page-comments"></a>
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-comments"><div class="sites-comment-docos-wrapper"><div class="sites-comment-docos"><div class="sites-comment-docos-background"></div><div class="sites-comment-docos-header"><div class="sites-comment-docos-header-title">Comments</div></div><div id="sites-comment-docos-pane" class="sites-comment-docos-pane"></div></div></div></div>
</div>
</div> 
</td> 
</tr>
</table> 
</div> 
</div> 
<div id="sites-chrome-footer-wrapper">
<div id="sites-chrome-footer-wrapper-inside">
<div id="sites-chrome-footer">
</div>
</div>
</div>
</div> 
</div> 
<div id="sites-chrome-adminfooter-container">
<div xmlns="http://www.w3.org/1999/xhtml" class="sites-adminfooter" role="navigation"><p><a class="sites-system-link" href="https://accounts.google.com/ServiceLogin?continue=https://sites.google.com/site/drrabhishekkumar/tools-and-softwares/regulatory-genomic-tools&amp;service=jotspot">Sign in</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/recentChanges.html">Recent Site Activity</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/reportAbuse.html" target="_blank">Report Abuse</a><span aria-hidden="true">|</span><a class="sites-system-link" href="javascript:;" onclick="window.open(webspace.printUrl)">Print Page</a><span aria-hidden="true">|</span><span class="sites-system-link">Powered By</span> <b class="powered-by"><a href="http://sites.google.com/site">Google Sites</a></b></p></div>
</div>
</div> 
</div> 
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('sjl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" src="../../../../ssl.gstatic.com/sites/p/6b8a44/system/js/jot_min_view__en.js"></script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('jl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
      gsites.HoverPopupMenu.createSiteDropdownMenus('sites-header-nav-dropdown', false);
    </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
            JOT_setupNav("2bd", "Navigation", true);
            JOT_addListener('titleChange', 'JOT_NAVIGATION_titleChange', 'COMP_2bd');
          </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
              new sites.CommentPane('//docs.google.com/comments/d/AAHRpnXvxeV4dz17kJ3ooY5tJ_dhOo25s0ZK3UMOxA2IEVPm9MWn5Kya1vgNTbr75j0Nt_7wrazMs8BxHXt-Li8Dzs3ldtmUHeuTkN75rR18Qt6KuB8HJV5-0wprpDEMBPO1Ah2qjYrwG/api/js?anon=true',
                  false, false);
            </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
  setTimeout(function() {
    var fingerprint = gsites.date.TimeZone.getFingerprint([1109635200000, 1128902400000, 1130657000000, 1143333000000, 1143806400000, 1145000000000, 1146380000000, 1152489600000, 1159800000000, 1159500000000, 1162095000000, 1162075000000, 1162105500000]);
    gsites.Xhr.send('../_/tz.html', null, null, 'GET', null, null, { afjstz: fingerprint });
  }, 500);
</script>
<script xmlns="http://www.w3.org/1999/xhtml">
                    window.onload = function() {
                      if (false) {
                        JOT_setMobilePreview();
                      }
                      var loadTimer = window.jstiming.load;
                      loadTimer.tick("ol");
                      loadTimer["name"] = "load," + webspace.page.type + ",user_page";
                      window.jstiming.report(loadTimer, {}, 'https://gg.google.com/csi');
                    }
                  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
        JOT_insertAnalyticsCode(false,
            false);
      </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    var maestroRunner = new gsites.pages.view.SitesMaestroRunner(
        webspace, "en");
    maestroRunner.initListeners();
    maestroRunner.installEditRender();
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
  //<![CDATA[
    // Decorate any fastUI buttons on the page with a class of 'goog-button'.
    if (webspace.user.hasWriteAccess) {
      JOT_decorateButtons();
    }

    // Fires delayed events.
    (function() {
      JOT_fullyLoaded = true;
      var delayedEvents = JOT_delayedEvents;
      for (var x = 0; x < delayedEvents.length; x++) {
        var event = delayedEvents[x];
        JOT_postEvent(event.eventName, event.eventSrc, event.payload);
      }
      JOT_delayedEvents = null;
      JOT_postEvent('pageLoaded');
    })();
  //]]>
</script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    JOT_postEvent('decorateGvizCharts');
  </script>
<script type="text/javascript">
          JOT_setupPostRenderingManager();
        </script>
<script type="text/javascript">
          JOT_postEvent('renderPlus', null, 'sites-chrome-main');
        </script>
<script type="text/javascript">
          sites.codeembed.init();
        </script>
<div id="server-timer-div" style="display:none"> </div>
<script type="text/javascript">
          window.jstiming.load.tick('render');
          JOT_postEvent('usercontentrendered', this);
        </script>
</body>

<!-- Mirrored from sites.google.com/site/drrabhishekkumar/tools-and-softwares/regulatory-genomic-tools by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 25 Feb 2019 18:04:28 GMT -->
</html>
