<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" itemscope="" itemtype="http://schema.org/WebPage">

<!-- Mirrored from sites.google.com/site/drrabhishekkumar/research-profile/research-experiences by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 25 Feb 2019 18:04:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="chrome=1" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { (function(){function e(a){this.t={};this.tick=function(a,c,b){this.t[a]=[void 0!=b?b:(new Date).getTime(),c];if(void 0==b)try{window.console.timeStamp("CSI/"+a)}catch(h){}};this.tick("start",null,a)}var a;if(window.performance)var d=(a=window.performance.timing)&&a.responseStart;var f=0<d?new e(d):new e;window.jstiming={Timer:e,load:f};if(a){var c=a.navigationStart;0<c&&d>=c&&(window.jstiming.srt=d-c)}if(a){var b=window.jstiming.load;0<c&&d>=c&&(b.tick("_wtsrt",void 0,c),b.tick("wtsrt_","_wtsrt",
d),b.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),b&&0<c&&(b.tick("_tbnd",void 0,window.chrome.csi().startE),b.tick("tbnd_","_tbnd",c))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,b&&0<c&&(b.tick("_tbnd",void 0,window.external.startE),b.tick("tbnd_","_tbnd",c))),a&&(window.jstiming.pt=a)}catch(g){}})(); })()
</script>
<link rel="shortcut icon" type="image/x-icon" href="http://www.google.com/images/icons/product/sites-16.ico" />
<link rel="apple-touch-icon" href="../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/images/apple-touch-icon.png" type="image/png" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { function d(a){return document.getElementById(a)}window.byId=d;function g(a){return a.replace(/^\s+|\s+$/g,"")}window.trim=g;var h=[],k=0;window.JOT_addListener=function(a,b,c){var f=new String(k++);a={eventName:a,handler:b,compId:c,key:f};h.push(a);return f};window.JOT_removeListenerByKey=function(a){for(var b=0;b<h.length;b++)if(h[b].key==a){h.splice(b,1);break}};window.JOT_removeAllListenersForName=function(a){for(var b=0;b<h.length;b++)h[b].eventName==a&&h.splice(b,1)};
window.JOT_postEvent=function(a,b,c){var f={eventName:a,eventSrc:b||{},payload:c||{}};if(window.JOT_fullyLoaded)for(b=h.length,c=0;c<b&&c<h.length;c++){var e=h[c];e&&e.eventName==a&&(f.listenerCompId=e.compId||"",(e="function"==typeof e.handler?e.handler:window[e.handler])&&e(f))}else window.JOT_delayedEvents.push({eventName:a,eventSrc:b,payload:c})};window.JOT_delayedEvents=[];window.JOT_fullyLoaded=!1;
window.JOT_formatRelativeToNow=function(a,b){a=((new Date).getTime()-a)/6E4;if(1440<=a||0>a)return null;var c=0;60<=a&&(a/=60,c=2);2<=a&&c++;return b?window.JOT_siteRelTimeStrs[c].replace("__duration__",Math.floor(a)):window.JOT_userRelTimeStrs[c].replace("__duration__",Math.floor(a))}; })()
</script>
<script>

  

  var breadcrumbs = [{"path":"/site/drrabhishekkumar/research-profile","deleted":false,"title":"Research Portfolio","dir":"ltr"},{"path":"/site/drrabhishekkumar/research-profile/research-experiences","deleted":false,"title":"Research Experiences","dir":"ltr"}];
  var JOT_clearDotPath = '../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/images/cleardot.gif';

  
  var JOT_userRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

  
  

  

  var webspace = {"gvizGstaticVersion":"current","enableAnalytics":false,"pageSharingId":"jotspot_page","codeembeds":{"outerIframeSrc":"https://www.gstatic.com/jotspot/embeds/code/0f08d42392f2000e7e3f3daf5b427a43/outer_iframe.html","innerIframeSrc":"https://1346992350-jotspot-embeds.googleusercontent.com/code/8d87fa64604b2a11fae2ed06104c58d3/inner_iframe.html"},"enableUniversalAnalytics":false,"sharingPolicy":"OPENED","siteTitle":"Dr. Abhishek Kumar","jot2atari":{"eligibility":"ineligible"},"onepickUrl":"https://docs.google.com/picker","adsensePublisherId":null,"features":{"oAuthForChartsApi":true,"moreMobileStyleImprovements":null,"googleChartsOverGstatic":false,"picasaAlbumInsert":false,"removeGooglePlusWidgets":true,"enableJot2Atari":true,"plusBadge":false},"isPublic":true,"newSitesBaseUrl":"https://sites.google.com","isConsumer":true,"serverFlags":{"jot2AtariLearnMoreUrl":"https://support.google.com/sites/answer/7035197"},"domainAnalyticsAccountId":"","plusPageId":"","signInUrl":"https://accounts.google.com/AccountChooser?continue\u003dhttps://sites.google.com/site/drrabhishekkumar/research-profile/research-experiences\u0026service\u003djotspot","analyticsAccountId":"","scottyUrl":"/_/upload","homePath":"/","siteNoticeUrlEnabled":null,"plusPageUrl":"","adsensePromoClickedOrSiteIneligible":true,"csiReportUri":"https://gg.google.com/csi","sharingId":"jotspot","termsUrl":"//www.google.com/intl/en/policies/terms/","gvizVersion":1,"editorResources":{"sitelayout":["https://ssl.gstatic.com/sites/p/6b8a44/system/app/css/sitelayouteditor.css"],"text":["https://ssl.gstatic.com/sites/p/6b8a44/system/js/codemirror.js","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/codemirror_css.css","../../../../ssl.gstatic.com/sites/p/6b8a44/system/js/trog_edit__en.js","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/trogedit.css","../_/rsrc/1550133439000/system/app/css/editor.css","../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/css/codeeditor.css","/site/drrabhishekkumar/_/rsrc/1550133439000/system/app/css/camelot/editor-jfk.css"]},"sharingUrlPrefix":"/_/sharing","isAdsenseEnabled":true,"domain":"defaultdomain","baseUri":"/site/drrabhishekkumar","name":"drrabhishekkumar","siteTemplateId":false,"siteNoticeRevision":null,"siteNoticeUrlAddress":null,"siteNoticeMessage":null,"page":{"isRtlLocale":false,"canDeleteWebspace":null,"isPageDraft":null,"parentPath":"/research-profile","parentWuid":"wuid:gx:28368d211dc65250","siteLocale":"en","timeZone":"America/Los_Angeles","type":"text","title":"Research Experiences","locale":"en","wuid":"wuid:gx:1b31dde1ae53a3a2","revision":2,"path":"/research-profile/research-experiences","isSiteRtlLocale":false,"pageInheritsPermissions":null,"name":"research-experiences","canChangePath":true,"state":"","properties":{},"bidiEnabled":false,"currentTemplate":{"path":"/system/app/pagetemplates/text","title":"Web Page"}},"canPublishScriptToAnyone":true,"user":{"keyboardShortcuts":true,"sessionIndex":"","onePickToken":"","guest_":true,"displayNameOrEmail":"guest","userName":"guest","uid":"","renderMobile":false,"domain":"","namespace":"","hasWriteAccess":false,"namespaceUser":false,"primaryEmail":"guest","hasAdminAccess":false},"gadgets":{"baseUri":"/site/drrabhishekkumar/system/app/pages/gadgets"}};
  webspace.page.breadcrumbs = breadcrumbs;

  
  var JOT_siteRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

</script>
<script type="text/javascript">
                window.jstiming.load.tick('scl');
              </script>
<meta name="title" content="Research Experiences - Dr. Abhishek Kumar" />
<meta itemprop="name" content="Research Experiences - Dr. Abhishek Kumar" />
<meta property="og:title" content="Research Experiences - Dr. Abhishek Kumar" />
<style type="text/css">
</style>
<link rel="stylesheet" type="text/css" href="../../../../ssl.gstatic.com/sites/p/6b8a44/system/app/themes/microsport/standard-css-microsport-ltr-ltr.css" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1550133439000/system/app/css/overlay4194.css?cb=microsport150goog-ws-nav-left-noheadnone30themedefaultstandard" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1550133439000/system/app/css/camelot/allthemes-view.css" />
<!--[if IE]>
          <link rel="stylesheet" type="text/css" href="/site/drrabhishekkumar/system/app/css/camelot/allthemes%2die.css" />
        <![endif]-->
<title>Research Experiences - Dr. Abhishek Kumar</title>
<meta itemprop="image" content="https://sites.google.com/site/draabhishekkumar/_/rsrc/1466376349730/reas/development-of-pipelines/research-experiences/Graphical_Abstract_porins.jpg?height=300&amp;width=400" />
<meta property="og:image" content="https://sites.google.com/site/draabhishekkumar/_/rsrc/1466376349730/reas/development-of-pipelines/research-experiences/Graphical_Abstract_porins.jpg?height=300&amp;width=400" />
<script type="text/javascript">
                window.jstiming.load.tick('cl');
              </script>
</head>
<body xmlns="http://www.google.com/ns/jotspot" id="body" class=" en            ">
<div id="sites-page-toolbar" class="sites-header-divider">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-status" class="sites-status" style="display:none;"><div id="sites-notice" class="sites-notice" role="status" aria-live="assertive"> </div></div>
</div>
<div id="sites-chrome-everything-scrollbar">
<div id="sites-chrome-everything" class="">
<div id="sites-chrome-page-wrapper" style="direction: ltr">
<div id="sites-chrome-page-wrapper-inside">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-chrome-header-wrapper" style="">
<table id="sites-chrome-header" class="sites-layout-hbox" cellspacing="0" style="">
<tr class="sites-header-primary-row" id="sites-chrome-userheader">
</tr>
<tr class="sites-header-secondary-row" id="sites-chrome-horizontal-nav">
<td colspan="2" id="sites-chrome-header-horizontal-nav-container" role="navigation">
<div class="sites-header-nav"><ul class="sites-header-nav-container-boxes"><li class="unselected"><a class="sites-navigation-link unselected" href="../home.html">Home</a></li></ul><div style="clear: both;"></div></div>
</td>
</tr>
</table>
</div>
<div id="sites-chrome-main-wrapper">
<div id="sites-chrome-main-wrapper-inside">
<table id="sites-chrome-main" class="sites-layout-hbox" cellspacing="0" cellpadding="{scmCellpadding}" border="0">
<tr>
<td id="sites-chrome-sidebar-left" class="sites-layout-sidebar-left initial" style="width:150px">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_2bd" class="sites-embed" role="navigation"><div class="sites-embed-content sites-sidebar-nav"><ul role="navigation" jotId="navList" class="has-expander"><li class="topLevel nav-first parent " wuid="gx:7181a2541346e1b0"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../home.html" jotId="wuid:gx:7181a2541346e1b0" class="sites-navigation-link topLevel">Home</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/awards-fellowships.html" jotId="wuid:gx:7b30df081f9ba5b9" class="sites-navigation-link">Awards &amp; Fellowships</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/cv.html" jotId="wuid:gx:739bc597c14732cb" class="sites-navigation-link">CV</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/education.html" jotId="wuid:gx:69568c90e4cd4346" class="sites-navigation-link">Education</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/lebenslauf.html" jotId="wuid:gx:9fbe71b7c99d1d8" class="sites-navigation-link">Lebenslauf</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../home/profile.html" jotId="wuid:gx:2db26263ae4fafb8" class="sites-navigation-link">Profile</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../bgiseq.html" jotId="wuid:gx:7c73f89c99b34613" class="sites-navigation-link topLevel">BGISEQ</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../collaborators.html" jotId="wuid:gx:daa1485044a82dd" class="sites-navigation-link topLevel">Collaborators</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../contact-me.html" jotId="wuid:gx:43b763280cdae9b1" class="sites-navigation-link topLevel">Contact Me</a></div></li><li class="topLevel parent " wuid="gx:9596e0d0a4c3aec"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../developments.html" jotId="wuid:gx:9596e0d0a4c3aec" class="sites-navigation-link topLevel">Developments</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/comparative-genomics-pipeline.html" jotId="wuid:gx:70d8b161a82ed9c0" class="sites-navigation-link">Comparative Genomics pipeline</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/pipeline-genetic-variants-and-phylogeny.html" jotId="wuid:gx:2c92bc6f3605332d" class="sites-navigation-link">Genetic variants and phylogeny</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/marine-fungal-genomics-pipeline.html" jotId="wuid:gx:d2a98c54427d68c" class="sites-navigation-link">Marine fungal genomics pipeline</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/proteogenomics-pipelines.html" jotId="wuid:gx:61bdfb4e8d513b78" class="sites-navigation-link">Proteogenomics pipelines</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../developments/insect-genomics-transcriptomics.html" jotId="wuid:gx:4cfd8657799fdfac" class="sites-navigation-link">Transcriptomic analyses pipeline</a></div></li></ul></li><li class="topLevel parent " wuid="gx:19dd9f1e0b68b591"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../hallmarks-of-genomics-bioinformatics.html" jotId="wuid:gx:19dd9f1e0b68b591" class="sites-navigation-link topLevel">Hallmarks of Genomics &amp; Bioinformatics</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../hallmarks-of-genomics-bioinformatics/nanopore.html" jotId="wuid:gx:1b0d3f710d95a120" class="sites-navigation-link">Nanopore</a></div></li></ul></li><li class="topLevel parent " wuid="gx:43d574afaf9eb7a1"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../job-search.html" jotId="wuid:gx:43d574afaf9eb7a1" class="sites-navigation-link topLevel">Job Search</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../job-search/jobs-in-germany.html" jotId="wuid:gx:1e8cd417bd04e76b" class="sites-navigation-link">Jobs In Germany</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../list-of.html" jotId="wuid:gx:465464df5caf5847" class="sites-navigation-link topLevel">List of Genomes</a></div></li><li class="topLevel parent " wuid="gx:9046d8ca199764a"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../living-in-germany.html" jotId="wuid:gx:9046d8ca199764a" class="sites-navigation-link topLevel">Living in Germany</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../living-in-germany/disability-and-life-in-germany.html" jotId="wuid:gx:3ed11e3d77709aac" class="sites-navigation-link">Disability and life in Germany</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../management-portfolio.html" jotId="wuid:gx:111bc9a268c7b7b8" class="sites-navigation-link topLevel">Management Portfolio</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../press-and-media.html" jotId="wuid:gx:5edb79290ac33e7b" class="sites-navigation-link topLevel">Press and Media</a></div></li><li class="topLevel parent " wuid="gx:6dd320bd7201d91c"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../professional-roles.html" jotId="wuid:gx:6dd320bd7201d91c" class="sites-navigation-link topLevel">Professional Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/editorial-board-member.html" jotId="wuid:gx:3ec0c9dfea1c55c8" class="sites-navigation-link">Editorial Board member</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/reviewer.html" jotId="wuid:gx:5ab4a629b1e809fa" class="sites-navigation-link">Reviewer</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../professional-roles/scientific-society-membership.html" jotId="wuid:gx:fc379cf6763ccee" class="sites-navigation-link">Scientific Society Membership</a></div></li></ul></li><li class="topLevel parent " wuid="gx:334fd11dea82d559"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../publications.html" jotId="wuid:gx:334fd11dea82d559" class="sites-navigation-link topLevel">Publication Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/books.html" jotId="wuid:gx:11b64f0213fc80c6" class="sites-navigation-link">Books</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/conference-posters.html" jotId="wuid:gx:512c5367927fe6f2" class="sites-navigation-link">Conference Posters</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/corresponding-author.html" jotId="wuid:gx:593aadc27dbbef19" class="sites-navigation-link">Corresponding Author</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/first-author.html" jotId="wuid:gx:64008ae6efa15d9d" class="sites-navigation-link">First Author</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/if-wise.html" jotId="wuid:gx:41d920a865179998" class="sites-navigation-link">IF-wise</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/list-of-posters-presented.html" jotId="wuid:gx:2dbbb3814063aadc" class="sites-navigation-link">List of posters presented</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/talks.html" jotId="wuid:gx:6bb48c5996447827" class="sites-navigation-link">Talks</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/top-publications.html" jotId="wuid:gx:5aebbe9a44453899" class="sites-navigation-link">Top Publications</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/top-listed.html" jotId="wuid:gx:65ef0b68b78e8942" class="sites-navigation-link">Top-Listed</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../publications/year-wise-publication-list.html" jotId="wuid:gx:6df2004a1dd59cb3" class="sites-navigation-link">Year wise publication list</a></div></li></ul></li><li class="topLevel parent " wuid="gx:28368d211dc65250"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../research-profile.html" jotId="wuid:gx:28368d211dc65250" class="sites-navigation-link topLevel">Research Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="junior-research-fellow-ncbs.html" jotId="wuid:gx:517c1f5fe5dd02b0" class="sites-navigation-link">Junior Research Fellow (NCBS)</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="msc-biotechnology.html" jotId="wuid:gx:26216082bbe83fd9" class="sites-navigation-link">MSc (Biotechnology)</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="phd-bioinformatics.html" jotId="wuid:gx:1115a41e1bbcca84" class="sites-navigation-link">PhD (Bioinformatics)</a></div></li><li class=""><div class="current-bg" jotId="wuid:gx:1b31dde1ae53a3a2" dir="ltr" style="padding-left: 38px;">Research Experiences</div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="research-interests.html" jotId="wuid:gx:34d09da5e6febfea" class="sites-navigation-link">Research interests</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../synteny-tools.html" jotId="wuid:gx:22f18bc3a7b2eef2" class="sites-navigation-link topLevel">Synteny tools</a></div></li><li class="topLevel parent " wuid="gx:30c77ffaa0a2a074"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../teaching.html" jotId="wuid:gx:30c77ffaa0a2a074" class="sites-navigation-link topLevel">Teaching Portfolio</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../teaching/python-programming.html" jotId="wuid:gx:220765ba8541ca5d" class="sites-navigation-link">Python Programming</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../teaching/training-on-reference-management-tools.html" jotId="wuid:gx:2c8565b3fd60619c" class="sites-navigation-link">Training on Reference Management Tools</a></div></li></ul></li><li class="topLevel parent " wuid="gx:326206f311529336"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../tools-and-softwares.html" jotId="wuid:gx:326206f311529336" class="sites-navigation-link topLevel">Tools and Softwares</a></div><ul role="navigation" class="has-expander"><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/all-sanger-tools.html" jotId="wuid:gx:689418979614cc74" class="sites-navigation-link">All Sanger tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/genetic-variant-analysis-tools.html" jotId="wuid:gx:439f968597b6f503" class="sites-navigation-link">Genetic Variant Analysis tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/genomics-tools.html" jotId="wuid:gx:5361d3e140dd53a9" class="sites-navigation-link">Genomics tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/missense-variants-ranking.html" jotId="wuid:gx:7316b4ab8208c56a" class="sites-navigation-link">Missense Variants Ranking</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/regulatory-genomic-tools.html" jotId="wuid:gx:767b5c2870871d3d" class="sites-navigation-link">Regulatory Genomic Tools</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/single-cell-genomics-transcriptomics.html" jotId="wuid:gx:7acb3cef3e39df30" class="sites-navigation-link">Single Cell Genomics &amp; Transcriptomics</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/structural-modelling-of-proteins.html" jotId="wuid:gx:1d9de95549bc3d48" class="sites-navigation-link">Structural modelling of proteins</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/tools-for-database-id-conversion.html" jotId="wuid:gx:7a739490df4e1b34" class="sites-navigation-link">tools for database id conversion</a></div></li><li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../tools-and-softwares/tools-for-pathway-analyses.html" jotId="wuid:gx:36848779604c56c7" class="sites-navigation-link">Tools for Pathway analyses</a></div></li></ul></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../useful-genomics-bioinformatics-links.html" jotId="wuid:gx:4f1fb032abe3227a" class="sites-navigation-link topLevel">Useful Genomics &amp; Bioinformatics links</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../variant2structure.html" jotId="wuid:gx:36befd3180c1797c" class="sites-navigation-link topLevel">Variant2Structure</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../videos.html" jotId="wuid:gx:8fe2237e3b0dd10" class="sites-navigation-link topLevel">Videos</a></div></li><li class="topLevel "><div dir="ltr" style="padding-left: 19px;"><a href="../system/app/pages/sitemap/hierarchy.html" jotId="wuid:gx:7700fb28d3d3353c" class="sites-navigation-link topLevel">Sitemap</a></div></li></ul></div></div>
</td>
<td id="sites-canvas-wrapper">
<div id="sites-canvas" role="main">
<div id="goog-ws-editor-toolbar-container"> </div>
<div xmlns="http://www.w3.org/1999/xhtml" id="title-crumbs" style="">
<A href="../research-profile.html" dir="ltr">Research Portfolio</A>‎ &gt; ‎
  </div>
<h3 xmlns="http://www.w3.org/1999/xhtml" id="sites-page-title-header" style="" align="left">
<span id="sites-page-title" dir="ltr" tabindex="-1" style="outline: none">Research Experiences</span>
</h3>
<div id="sites-canvas-main" class="sites-canvas-main">
<div id="sites-canvas-main-content">
<table xmlns="http://www.w3.org/1999/xhtml" cellspacing="0" class="sites-layout-name-one-column sites-layout-hbox"><tbody><tr><td class="sites-layout-tile sites-tile-name-content-1"><div dir="ltr"><div dir="ltr"><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">M. Sc.</span></u></b></font></p><p style="line-height:normal"><font size="3"><span style="font-family:Times New Roman">After
 completing my B.Sc. with subjects Zoology, Botany and Chemistry, I 
studied and gain a year of experience in at School of Environmental 
Sciences, Jawaharlal Nehru University, New Delhi and learned about basic
 and advanced theories and practical associated with En</span></font><span style="font-family:Times New Roman;font-size:medium">vironm</span><span style="font-family:Times New Roman;font-size:medium">ent
 Science and Ecology. Then, I was very fortunate to continue my 
post-graduate study in Biotechnology at School of Biotechnology, Madurai
 Kamaraj University, a premier research institution in India. The 
various theoretical and practical courses that I have been involved in 
here have equipped me to enter the world of scientific research with 
poise and confidence. At the end of first year M. Sc, I too</span><span style="font-family:Times New Roman;font-size:medium">k up a summer project entitled </span><b style="font-family:Times New Roman;font-size:medium"><i>"The Thermal Effects on Biomolecular Systems: Molecular Modeling and Simulational Studies"</i></b><span style="font-family:Times New Roman;font-size:medium"> at
 Centre for Biotechnology, Jawaharlal Nehru University, New Delhi, India
 in Molecular Modeling of proteins. I examined the effect of temperature
 on proteins by using molecular modeling &amp; simulational studies. 
This was a novel experience and I really enjoyed mastering the different
 computational approaches of molecular modeling. In December 2000 
through an online test, I got opportunity to work in Computational 
Neurobiology at National Centre for Biological Sciences, Bangalore, 
India. I did a small project entitled as </span><b style="font-family:Times New Roman;font-size:medium"><i>“Modeling of Extracellular signal-Regulated Kinase (ERK) Signaling Pathways in Neurons”</i></b><span style="font-family:Times New Roman;font-size:medium"> that
 gave me idea how to model a signaling pathway and its kinetics &amp; 
how to solve a problem related to that pathway. I was involved in an 
independent research project entitled </span><b style="font-family:Times New Roman;font-size:medium"><i>"Structural analysis of outer membrane beta-stranded porins using Temperature factor"</i> </b><span style="font-family:Times New Roman;font-size:medium">at Bioinformatics Centre, Madurai Kamaraj University, India.</span></p><img border="0" height="300" src="../../draabhishekkumar/_/rsrc/1466376349730/reas/development-of-pipelines/research-experiences/Graphical_Abstract_porinsf0aa.jpg?height=300&amp;width=400" style="font-size:10pt;line-height:1.6;display:inline;float:left;margin:5px 10px 0px 0px" width="400" /><p style="line-height:normal"><font size="3"><span style="font-family:Times New Roman"> In
 this project, I wrote scripts for calculating average atomic 
temperature factors (B-values) and its standard deviations for different
 secondary structures at atom type level and residue type level of high 
resolution crystal structures of porins and found that inner wall of 
porins that face towards trimmer interface has lower average B-values 
than that of outer wall and residue distributions in all porins is quite
 different </span><span style="line-height:18.399999618530273px;font-family:Times New Roman">different <font color="#ff0000"><b>(Kumar &amp; Krishnaswamy (2008), </b></font></span><a href="../../draabhishekkumar/reas/publications.html"><font color="#ff0000"><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">Nature Precedings</span><span style="line-height:18.399999618530273px;font-family:Times New Roman">)</span></b></font></a><span style="line-height:18.399999618530273px;font-family:Times New Roman">.</span></font></p><p style="margin-left:18pt;line-height:normal"><span style="font-family:Times New Roman;font-size:medium;line-height:18px">Recently a re</span><span style="font-family:Times New Roman;font-size:medium;line-height:18px">view on porin became available as </span><span style="font-family:Times New Roman;font-size:medium;line-height:25px"><b><font color="#ff0000">(Kumar </font><i style="color:red">et al. </i><font color="#ff0000">(2014), </font><a href="http://eurekaselect.com/124220" rel="nofollow" target="_blank"><font color="#ff0000">Protein and Peptide Letters</font></a><font color="#ff0000">).</font></b></span></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">Junior Research Fellow at National Centre for Biological Sciences, India</span></u></b></font></p><p style="line-height:normal"><span style="font-family:Times New Roman"><font size="3"><br />I worked as Junior Research Fellow at National Centre for Biological Sciences, Bangalore, India with project entitled <b><i>"Recognition of folds and functions of protein domain superfamilies coded in genome sequences."</i></b> funded by <b>Wellcome Trust</b>,
 London. The main objective of the project was to apply the information 
available in the superfamily alignment database for fold and/or function
 prediction of newly determined protein sequences, in addition to study <i>Drosophila</i> olfactory receptors to order to understand how olfaction happens in<b> </b><i>Drosophila</i> in collaboration with Tata Institute of Fundamental Research, Mumbai, India<b>.</b> I analyzed Sequences of <i>Drosophila</i> olfactory
 receptors after taking it from Fly-base database and compared with 
mammalian olfactory receptors. I derived good alignment of these 
sequences after rigorous alignments, predicted 7 trans-membrane regions 
of these sequences and build on first ever possible models of <i>Drosophila </i>olfactory Receptors. Subsequently, I examined to find how <i>Drosophila</i> olfactory receptors help in axonal guidance and I analyzed tails of <i>Drosophila</i> olfactory receptors and found a specific charge pattern generally available in other receptors involve in axonal guidance.</font></span></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">Ph.D. - “<i>Phylogenomics of vertebrate serpins</i>”</span></u></b></font></p><div><p style="line-height:normal"><font size="3"><span lang="EN-GB" style="font-family:Times New Roman">I worke</span></font><font size="3"><span lang="EN-GB" style="font-family:Times New Roman">d
 at the University of Bielefeld as Ph.D. researcher and I have been 
dealing with comparative and phylogenetic analysis of serine proteinase 
inhibitors (serpins). The serpins constitute a superfamily of proteins 
that fold into a conserved tertiary structure and employ a 
sophisticated, irr</span></font><font size="3"><span lang="EN-GB" style="font-family:Times New Roman">eversible
 suicide-mechanism of inhibition. More than 500 serpins have been 
identified, occurring in all three forms of the life- the eukaryotes, 
the prokaryotes and the archea. </span><span style="font-family:Times New Roman">Vertebrate
 serpins can be conveniently classified into six groups, V1-V6, based on
 three independent biological features - genomic organization, 
diagnostic amino acid sites and rare indels. The present vertebrate 
serpins are derived from an original serpin most probably by intron 
insertion. </span><span lang="EN-GB" style="font-family:Times New Roman">I have investigated the phylogenetic relationship of serpin from </span><em><span style="font-family:Times New Roman">Nematostella vectensis, Strongylocentrotus purpuratus, Drosophila melanogaster</span></em><b><i><span lang="EN-GB" style="font-family:Times New Roman">, </span></i></b><i><span lang="EN-GB" style="font-family:Times New Roman">Ciona intestinalis, </span></i><span lang="EN-GB" style="font-family:Times New Roman">six
 fish species, frog, chicken and mammals using gene architecture 
analyses and stringent criteria for identification of orthologs. All 
vertebrate serpin genes fit into one of the six exon/intron gene classes
 previously identified with minor deviations, dating the existence and 
maintenance of these gene organiz</span></font><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman">ations
 before or close to the divergence of fishes. Group V1 and V2 gene 
families underwent rapid adaptive radiation along the lineages leading 
to mammals as indicated by an up to nine-fold increase of family 
members, accompanied by a rapid functional diversification. In contrast,
 gene groups V3 to V6 display a rather conservative evolution with 
little changes since the divergence of fishes and the other vertebrates.
 There is no indication for a duplication of serpins genes indicative of
 fish-specific whole genome duplication events. Alternatively duplicated
 genes may have been lost. The orthology assessment indicates that all 
vertebrates are equipped with a subset of strongly conserved serpins 
with functions that can be clearly correlated with basic 
vertebrate-specific physiology.</span><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman"> </span></p><p style="line-height:normal"><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman">Amongst
 the vertebrate serpins the genomic organisation and corresponding 
syntenies between different genomes were also analysed </span></p><img border="0" height="400" src="../../../../www.biomedcentral.com/content/figures/1471-2148-8-250-2-l.html" style="font-size:10pt;line-height:1.6;display:inline;margin:5px 10px;float:right" width="223" /><p style="line-height:normal"><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman">and
 the result show massive gene duplication, gene diversification, and 
gene gain and loss during vertebrate evolution over 450 Mya. The 
urochordates and invertebrates do not share even a single genome synteny
 level conservation.</span><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman"> </span><span lang="EN-GB" style="font-size:medium;font-family:Times New Roman">None of <i>Ciona intestinalis</i> serpin
 genes shares a common exon-intron architecture organisation with any of
 the vertebrate serpin gene classes, nor was it possible to identify 
orthologs of vertebrates. The lack of gene architecture similarity and 
the complete absence of orthology between urochordate and vertebrate 
serpin indicate that major changes with bursts of character acquisition 
must have occurred during evolution of serpins in the time interval 
separating urochordates from chordates, indicating massive intron 
gain/loss and events providing C and N-terminal sequence extensions 
characteristic for today’s vertebrate serpins.</span></p><p style="line-height:normal"><span lang="EN-GB" style="font-family:Times New Roman"><font size="3">On
 unravelling the comparative genomics of serpins into evolutionary 
sense, I found two major outcomes of research work as following:</font></span></p><p style="text-indent:0cm;margin-left:0cm;line-height:normal"><font size="3"><span lang="EN-GB" style="font-family:Times New Roman">R</span><span style="font-family:Times New Roman">are genomic characters are used to show that orthologs of neuroserpin, a prominent representative of vertebrate group V3 <em>serpin </em>genes,
 exist in early diverging deuterostomes and probably also in cnidarians,
 indicating that the origin of a mammalian serpin can be traced back far
 in the history of eumetazoans. A C-terminal address code assigning 
association with secretory pathway organelles is present in all 
neuroserpin orthologs, suggesting that supervision of cellular 
export/import routes by antiproteolytic serpins is an ancient trait <a href="../../draabhishekkumar/reas/publications.html"> <span style="color:red"><b>(Kumar &amp; Ragg (2008), BMC Evolutionary Biology)</b></span>.</a></span></font></p><p style="text-indent:0cm;margin-left:0cm;line-height:normal"><span style="font-size:medium"><span style="font-family:Times New Roman"> </span></span><img border="0" height="263" src="../../../../www.biomedcentral.com/content/figures/1471-2148-9-208-1-l.html" style="line-height:1.6;font-size:10pt;display:inline;float:left;margin:5px 10px 0px 0px" width="400" /><span style="font-family:Times New Roman;font-size:medium;text-indent:0cm"><br /></span></p><p style="text-indent:0cm;margin-left:0cm;line-height:normal"><span style="text-indent:0cm;font-family:Times New Roman;font-size:medium">After
 establishment of canonical exon-intron patterns in the serpins 
superfamily at the dawn of vertebrate evolution, multiple intron 
acquisition events have occurred during diversification of a lineage of 
actinopterygian fishes. The novel introns were gathered within a limited
 time interval (on an evolutionary timescale), and no such events were 
observed in other groups of vertebrates. Examination of the sequences 
flanking the intron insertion points revealed that the genetic 
requirements for acquisition of novel introns may be less stringent than
 previously suggested. Finally, we argue that genome compaction, a 
phenomenon associated with the fish lineage depicting preferential 
intron gain, might promote intron gain </span><a href="../../draabhishekkumar/reas/publications.html" style="text-indent:0cm;font-family:Times New Roman;font-size:medium" target="_blank"><font color="#ff0000"><b>(Ragg &amp; Kumar <i>et al</i>. (2009), BMC Evolutionary Biology)</b></font></a><span style="text-indent:0cm;font-family:Times New Roman;font-size:medium">.</span></p><p style="margin-left:18pt;line-height:normal"><span style="font-family:Times New Roman"><font size="3"> </font></span></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol"><br /></span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">Understanding mechanisms of human diseases</span></u></b></font></p><p style="line-height:normal"><span style="font-family:Times New Roman"><font size="3">During
 this Ph.D. period, I gained experiences in evolutionary genomics and 
biology and my quest for applying comparative genomics approaches to 
different medicinal issues enormously got enhanced as I realized that 
comparative approaches could unravel newer understandings into different
 human diseases and disorders. To carry on further, I have been involved
 into two different projects as described below:</font></span></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">Unraveling novel human kinases and their potential roles in human diseases such as cancer</span></u></b></font></p><p style="line-height:normal"><font size="3"><span style="font-family:Times New Roman"> </span><span style="font-family:Times New Roman">In
 collaboration, cardiovascular research group in University of Munich 
(LMU), I found that the previously published coding sequence of the <i>STK35</i> gene is incomplete. The newly identified <i>STK35</i> gene
 codes for a protein of 534 amino acids. This protein (but not 
STK35/Clik1) was found to be expressed in all human cells studied 
(endothelial cells, Macrophages, HeLa and HEK cells) EGFP-STK35L was 
localized in the nucleus and the nucleolus. Hence I termed this protein 
as STK35L1 (<u>STK35</u> <u>L</u>ong). By combining syntenic and gene structure pattern data and homology searches, a new <i>STK35L</i> protein kinase sub-family<i> </i>consisting of three members (<i>STK35L1</i>, <i>STK35L2</i>, and <i>STK35L3</i>) was identified. Interestingly, all these members were conserved throughout the vertebrates. However, <i>STK35L3</i> was
 specifically lost during placental mammalian evolution. Using 
comparative genomics, we have identified orthologous sets of these three
 genes and their possible ancestor gene in two sea squirt genomes. We 
have characterized a new subfamily of kinases termed as ‘<i>STK35L </i>subfamily’. The localization of <i>STK35L1</i> in cells suggests a functional role in the nucleolus <a href="../../draabhishekkumar/reas/publications.html" target="_blank"><font color="#ff0000"><b>(Goyal <i>et al. </i>(2009), PLoS ONE and Goyal <i>et al.</i> (2011), PLoS ONE)</b></font></a>. This study will give a platform to explore the role of <i>STK35L</i> kinases in cell functions and human diseases.</span></font></p><p style="margin-left:18pt;line-height:normal"><font size="3"><span style="font-family:Times New Roman">  </span><b><u><span style="font-family:Times New Roman">Comparative genomics of cerebral cavernous malformations (CCM) and unraveling pathways in different model organisms</span></u></b></font></p><p style="line-height:normal"><font size="3"><span style="font-family:Times New Roman">The three cerebral cavernous malformations (CCM) genes namely </span><span style="font-family:Times New Roman">CCM1 (KRIT1), CCM2 (MGC4607), and CCM3 (PDCD10)</span><span style="font-family:Times New Roman"> have
 been identified, which by mutation cause cerebral cavernous 
malformations, a common deadly disease in humans. Defining the molecular
 phylogeny of CCM genes by using comparative genomics tools is one 
approach to understand the different links amongst their gene structure,
 synteny organization and their function in these biological processes, 
which are not fully understood yet. The CCM genes first appear in 
invertebrates and our studies suggest that the regulation and biological
 actions of this gene are preserved across invertebrates to vertebrates.
 This comprehensive analysis of the CCM genes across phyla provides new 
information that will aid further characterization of these CCM genes in
 their biological processes which will let new era of understanding in 
the cerebral cavernous malformations as a disease and their medicinal 
aspect. Understanding basic pathway in different model organisms is 
necessary in order to provide answers of unsolved mechanisms controlling
 this disease.</span></font></p><p style="margin:0cm 0cm 0.0001pt 17.85pt;line-height:normal"><font size="3"><span style="font-family:Symbol">·<span style="font-family:Times New Roman">      </span></span><b><u><span style="font-family:Times New Roman">Comparative genomics of GPCRs such as </span></u></b><b><u><span style="font-family:Times New Roman"> lysophosphatidic acid receptors (LPARs) and melanocortin receptors (MCR) </span></u></b></font></p><p style="margin-bottom:0.0001pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Times New Roman"> </span></font></p><img border="0" height="274" src="../../draabhishekkumar/_/rsrc/1466376349731/reas/development-of-pipelines/research-experiences/journal.pone.0022046.g002cb67.png?height=274&amp;width=400" style="display:inline;float:right;margin:5px 10px" width="400" /><p style="margin-bottom:0.0001pt"><font size="3"><span style="font-family:Times New Roman">G</span></font><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">PCRs
 forms largest superfamily of transmembrane proteins and these serve as 
&gt;30% of pharmaceutically important receptors. Herein, I studied the 
comparative genomics of </span><i style="font-family:Times New Roman;font-size:medium;line-height:1.6">- </i><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">lysophosphatidic acid receptor</span><i style="font-family:Times New Roman;font-size:medium;line-height:1.6"> </i><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">(LPA) receptors and melanocortin (MC) receptors.</span><b style="font-family:Times New Roman;font-size:medium;line-height:1.6"> </b><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">I
 unraveled the molecular phylogeny of these GPCRs and I found that these
 genes have their own evolutionary history such as gain /loss of novel 
introns in genome compacted fishes. Part of results, I have published 
as </span><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0022046" rel="nofollow" style="font-family:Times New Roman;font-size:medium;line-height:1.6" target="_blank"><b><font color="#ff0000">Kumar et al. (2011, PLoS ONE)</font></b>.</a></p><p style="margin-bottom:0.0001pt"><span style="font-family:Times New Roman"><font size="3"><br /></font></span></p><p style="margin-bottom:0.0001pt"><br /></p><p style="margin:0cm 0cm 0.0001pt 18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Symbol">·<span style="line-height:normal;font-family:Times New Roman">      </span></span><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">Molecular Phylogeney of OVO-like transcription factors</span></b></font></p><p style="margin:0cm 0cm 0.0001pt 18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Times New Roman">  </span></font></p><img border="0" height="301" src="../../draabhishekkumar/_/rsrc/1466376349731/reas/development-of-pipelines/research-experiences/journal.pone.0039399.g008ea2d.png?height=301&amp;width=400" style="font-size:10pt;line-height:1.6;display:inline;float:left;margin:5px 10px 0px 0px" width="400" /><p style="margin:0cm 0cm 0.0001pt 18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Times New Roman"> 
              I designed and carried out molecular phylogeny of OVO-like
 transcription factors. It was reported that a core domain is </span><span style="font-family:Times New Roman;line-height:18.399999618530273px">c</span></font><span style="font-family:Times New Roman;line-height:18.399999618530273px;font-size:medium">onserved </span></p><p style="margin:0cm 0cm 0.0001pt 18pt"><font size="3"><span style="font-family:Times New Roman;line-height:18.399999618530273px">fr</span></font><span style="font-family:Times New Roman;line-height:18.399999618530273px;font-size:medium">om early-branching metazoa to humans and it is comprising a tetrad of C2H2 zinc fingers. By domain </span><span style="font-size:medium;font-family:Times New Roman;line-height:18.399999618530273px">comparison of the OVOL proteins, this study found that they evolved in different metazoan lineages by attaching </span><span style="font-size:medium;font-family:Times New Roman;line-height:18.399999618530273px">intrinsically-disordered (ID) segments of N/C-terminal extensions of 100 to 1000 amino acids to this conserved core.</span></p><p style="margin:0cm 0cm 0.0001pt 18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Times New Roman">These ID region</span></font><span style="font-size:medium;line-height:18.399999618530273px;font-family:Times New Roman">s originated independently across different animal lineages giving rise to different types of OVOL genes </span><span style="font-size:medium;font-family:Times New Roman;line-height:18.399999618530273px">over the course of metazoan evolution. This analysis illustrated the molecular evolution of metazoan OVOL genes over a </span><span style="font-size:medium;font-family:Times New Roman;line-height:18.399999618530273px">period of 700 million years (MY). This study both extends our current understanding of the structure/function relationship </span><span style="font-size:medium;font-family:Times New Roman;line-height:18.399999618530273px">of metazoan OVOL genes, and assembles a good platform for further characterization of OVOL genes from diverged </span><span style="font-size:medium;line-height:18.399999618530273px;font-family:Times New Roman">organisms <font color="#ff0000"><b>(</b></font></span><font color="#ff0000" style="font-size:medium;line-height:1.6"><b><span style="line-height:1.6;font-family:Times New Roman">Kumar <i>et al.</i> (2012), PLoS ONE</span><span style="line-height:1.6;font-family:Times New Roman">)</span></b></font><span style="font-size:medium;line-height:18.399999618530273px;font-family:Times New Roman">.</span></p><p style="margin-bottom:0.0001pt"><span style="font-family:Times New Roman"><font size="3"> </font></span></p><p style="margin-bottom:0.0001pt"><span style="font-family:Times New Roman"><font size="3"><br /></font></span></p><p style="margin-bottom:0.0001pt"><span style="font-family:Times New Roman"><font size="3"><br /></font></span></p><div style="display:inline;margin:5px 10px;float:right"><font size="3"><a href="../../../../f742e138-a-62cb3a1a-s-sites.googlegroups.com/site/draabhishekkumar/reas/development-of-pipelines/research-experiences/Slide122e7.jpg?attredirects=0" imageanchor="1"><img border="0" height="300" src="../../draabhishekkumar/_/rsrc/1466376349730/reas/development-of-pipelines/research-experiences/Slide1f0aa.jpg?height=300&amp;width=400" width="400" /></a></font></div><font size="3">·<span style="line-height:normal;font-family:Times New Roman">      </span><b><u><span style="line-height:18.399999618530273px;font-family:Times New Roman">Analyses of blood-related disease causing serpins</span></u></b></font><p style="margin-left:0cm"><span style="font-family:Times New Roman"><font size="3"><br /></font></span></p><p style="margin-left:0cm"><span style="font-family:Times New Roman"><font size="3">Recently,
 I designed and carried out researches of important blood-related 
disease-causing serpins for their genetic variants, mutational, 
evolutionary and structural perspectives and I have published on 
blood-pressure regulating angiotensinogen <span style="color:red"><b>(Kumar <i>et al. </i>(2014a), BBRC)</b></span>, blood clotting regulators such as antithrombin <span style="color:red"><b>(Kumar <i>et al. </i>(2013a), BBRC)</b></span><i> </i>and heparin cofactor II <span style="color:red"><b>(Kumar <i>et al. </i>(2013), Immunobiology)</b><i> </i></span>and plasma-regulator C1 inhibitors<b> <span style="color:red">(Kumar <i>et al. </i>(2014b), BBRC)</span></b>. </font></span></p><p style="margin-bottom:0.0001pt"><span style="font-family:Times New Roman"><font size="3"> </font></span></p><p style="margin-bottom:0.0001pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Times New Roman"> </span></font></p><p style="margin-left:18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Symbol">·<span style="line-height:normal;font-family:Times New Roman">      </span></span><span style="line-height:18.399999618530273px;font-family:Times New Roman"> <b><u>Bioinformatic postdoc at University of Padova, Italy</u></b></span></font></p><p style="margin-left:18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Courier New">o<span style="line-height:normal;font-family:Times New Roman">   </span></span><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">Sequencing, assembly and annotation of Colorado potato beetle transcriptome for pest management and population genetic analyses </span></b></font></p><img border="0" height="400" src="../../draabhishekkumar/_/rsrc/1466376349731/reas/development-of-pipelines/research-experiences/journal.pone.0086012.g0071611.png?height=400&amp;width=298" style="display:inline;margin:5px 10px;float:right" width="298" /><p style="margin-left:18pt"><font size="3"><b><span style="line-height:18.399999618530273px;font-family:Times New Roman"><span style="font-weight:normal;line-height:normal">         </span></span></b><span style="font-family:Times New Roman">The Colorado potato beetle (<i>Leptinotarsa. decemilineata</i>)
 is major pest poising serious threats to potato cultivation throughout 
the northern hemisphere. Despite its high-impact in invasive biology, 
phenology to pest management, little is known about L. decemilineata 
from a genomic perspective. Hence, we subjected the European <i>L. decemilineata</i> transcriptome to 454-FLX massively parallel DNA sequenc</span></font><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">ing </span><span style="font-family:Times New Roman;font-size:medium;line-height:1.6">in order to characterize diapause-specific genes and genes involved in pesticides and Bt resistance</span><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0086012" rel="nofollow" style="font-family:Times New Roman;font-size:medium;line-height:1.6" target="_blank"> <font color="#ff0000"><b>(Kumar et al. (2014),PLoS ONE).</b></font></a></p><p style="margin-left:18pt"><span style="font-family:Times New Roman"><font size="3"> Furthermore,
 I am also participating genome sequencing and analyses of this beetle 
under 5000 insect genome sequencing project with involvements of 
scientific community.</font></span></p><p><span lang="EN-GB"><font size="3"> </font></span></p><p style="margin-left:18pt"><font size="3"><span lang="EN-GB" style="font-family:Courier New">o<span style="line-height:normal;font-family:Times New Roman">   </span></span><b><span lang="EN-GB">Transcriptome sequencing and de novo annotation of the critically endangered Adriatic sturgeon</span></b></font></p><p><font size="3"><span lang="EN-GB">I also participated in the critically endangered Adriatic sturgeon, <i>Acipenser naccarii</i> </span><span lang="EN-GB" style="color:red"><b>(Vidotto<i> et al</i>. (2012), BMC Genomics)</b></span><span lang="EN-GB">.</span></font></p><p><span lang="EN-GB"><font size="3"> </font></span></p><p style="margin-left:0cm"><b><u><span style="line-height:15.333332061767578px;font-family:Times New Roman"><font size="3"> </font></span></u></b></p><p style="margin-left:0cm"><font size="3"><span style="line-height:18.399999618530273px;font-family:Symbol">·<span style="line-height:normal;font-family:Times New Roman">      </span></span><b><u><span style="line-height:18.399999618530273px;font-family:Times New Roman">Research Associate at University of Kiel</span></u></b></font></p><p style="margin-left:18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Courier New">o<span style="line-height:normal;font-family:Times New Roman">   </span></span><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">Genome sequencing, assembly and annotation of marine fungi for bio- and chemical diversity</span></b></font></p><p style="margin-left:0cm"><span style="font-family:Times New Roman"><font size="3">It
 is estimated that there are several hundreds of marine fungi species 
and they possess unique features in the fungal diversity, as they have 
to adopt marine environment. The enormous biodiversity of marine fungi 
is mirrored by the molecular diversity of their secondary metabolites. 
However, a very little is known about marine fungal genetic resources. 
Although several terrestrial habitant fungal genomes have been sequenced
 using Sanger sequencing in last two decades and recently, the <i>Sordaria macrospora</i> genomic
 sequencing became available using next-generation sequencing. However, 
not a single genome of marine isolates of fungi is yet known. I have 
established the genome and transcriptomes of three marine fungi (namely <i>Scopulariopsis brevicaulis</i>, <i>Pestalotiopsis</i>and <i>Calcarisporium</i>)
 using three different next-generation sequencing methods namely, Roche 
454, Illumina and Ion torrent. This genome characterization assists 
mycologist to further carry out research with this species, which 
largely hindered due to unavailability of the genome. There are four 
manuscripts in preparation from these research works. Seven potential 
drug-like molecules are identified during this work, including one 
anti-cancerous bioactive molecule. These molecules are under 
consideration to sell to pharmaceutical companies.</font></span></p><p style="margin-left:0cm"><b><span style="line-height:15.333332061767578px;font-family:Times New Roman"><font size="3"> </font></span></b></p><p style="margin-left:18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Courier New">o<span style="line-height:normal;font-family:Times New Roman">   </span></span><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">Studying unknown gene bem46 from <i>Neurospora crassa</i></span></b></font></p><p style="margin-left:0cm;line-height:normal"><span style="font-family:Times New Roman"><font size="3">Gene
 annotation and characterizations are still challenging tasks. These 
lead into problems for genome sequencing projects and about 40-50% of 
genes in genome remains unknown till today. Some of these genes are 
widespread conserved and termed as “known unknown“ and <i>bem46</i> gene
 is such a gene, which belongs the alpha/beta-hydrolase superfamily. In 
desktop genomic era, it is critical to find solution for large dataset 
that remain unknown. Recently, we demonstrated how conserved BEM46 
homologs are evolved by using exonic indel during eukaryotic evolution. 
The gene strucutures of BEM46 homologs are hypervarible with one exon in
 yeast to thirteen exon in human. We illustrated the catalytic sites of 
BEM46 <i>Neurospora crassa</i> as a catalytic triad formed by a serine 188S, a histidine 292H and an aspartic acid 262D. We also characterized <i>bem46</i> gene from <i>N. crassa</i> for localization studies and we defined new ER-retention signal “PEKK”, which was previously not described <span style="color:red"><b>(Kumar <i>et al. </i>(2013b), BBRC)</b></span>.
 Such studies are urgent requirements in modern biosciences to study new
 genes. Otherwise, we will remain puzzling about half of genes in newly 
sequenced genomes using rapid sequencing technologies.</font></span></p><p style="margin-left:0cm;line-height:normal"><span style="font-family:Times New Roman"><font size="3"> </font></span></p><p style="margin-left:18pt"><font size="3"><span style="line-height:18.399999618530273px;font-family:Courier New">o<span style="line-height:normal;font-family:Times New Roman">   </span></span><b><span style="line-height:18.399999618530273px;font-family:Times New Roman">RNA secondary structure changes with RNA editing events</span></b></font></p><p style="margin-left:0cm"><span style="font-family:Times New Roman"><font size="3">The
 maize cox2 RNA is fully edited in cauliflower mitochondria. Deletions 
in the cox2 RNA sequence can have a strong effect on RNA processing, 
leading to loss of splicing, loss of editing at all sites, or even to a 
gain of new editing sites. As these effects are not limited to the 
vicinity of the respective deletions, but appear to be widespread or 
even affect all editing sites, they may not be explained by the loss of 
PPR binding sites. Instead, it appears that several parts of the cox2 
transcript are required for proper RNA processing. This indicates the 
roles of the RNA sequence and structural elements in the recognition of 
the editing sites <span style="color:red"><b>(Gruettner<i> et al. </i>(2014), PLoS ONE)</b></span>.</font></span></p></div><div><span style="font-family:Times New Roman"><font size="3"><br /></font></span></div></div></div></td></tr></tbody></table>
</div> 
</div> 
<div id="sites-canvas-bottom-panel">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-subpages"> </div>
<div id="sites-attachments-container">
</div>
<a xmlns="http://www.w3.org/1999/xhtml" name="page-comments"></a>
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-comments"><div class="sites-comment-docos-wrapper"><div class="sites-comment-docos"><div class="sites-comment-docos-background"></div><div class="sites-comment-docos-header"><div class="sites-comment-docos-header-title">Comments</div></div><div id="sites-comment-docos-pane" class="sites-comment-docos-pane"></div></div></div></div>
</div>
</div> 
</td> 
</tr>
</table> 
</div> 
</div> 
<div id="sites-chrome-footer-wrapper">
<div id="sites-chrome-footer-wrapper-inside">
<div id="sites-chrome-footer">
</div>
</div>
</div>
</div> 
</div> 
<div id="sites-chrome-adminfooter-container">
<div xmlns="http://www.w3.org/1999/xhtml" class="sites-adminfooter" role="navigation"><p><a class="sites-system-link" href="https://accounts.google.com/ServiceLogin?continue=https://sites.google.com/site/drrabhishekkumar/research-profile/research-experiences&amp;service=jotspot">Sign in</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/recentChanges.html">Recent Site Activity</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/reportAbuse.html" target="_blank">Report Abuse</a><span aria-hidden="true">|</span><a class="sites-system-link" href="javascript:;" onclick="window.open(webspace.printUrl)">Print Page</a><span aria-hidden="true">|</span><span class="sites-system-link">Powered By</span> <b class="powered-by"><a href="http://sites.google.com/site">Google Sites</a></b></p></div>
</div>
</div> 
</div> 
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('sjl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" src="../../../../ssl.gstatic.com/sites/p/6b8a44/system/js/jot_min_view__en.js"></script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('jl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
      gsites.HoverPopupMenu.createSiteDropdownMenus('sites-header-nav-dropdown', false);
    </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
            JOT_setupNav("2bd", "Navigation", true);
            JOT_addListener('titleChange', 'JOT_NAVIGATION_titleChange', 'COMP_2bd');
          </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
              new sites.CommentPane('//docs.google.com/comments/d/AAHRpnXvxeV4dz17kJ3ooY5tJ_dhOo25s0ZK3UMOxA2IEVPm9MdHK4tR0TjPLjXUCf9w832wfktTw50XXNBMOkgA3dXVLv60uwI1HdFAMTogBAWYmVIRsqIOa9U166_pmaPg2BAGBaoji/api/js?anon=true',
                  false, false);
            </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
  setTimeout(function() {
    var fingerprint = gsites.date.TimeZone.getFingerprint([1109635200000, 1128902400000, 1130657000000, 1143333000000, 1143806400000, 1145000000000, 1146380000000, 1152489600000, 1159800000000, 1159500000000, 1162095000000, 1162075000000, 1162105500000]);
    gsites.Xhr.send('../_/tz.html', null, null, 'GET', null, null, { afjstz: fingerprint });
  }, 500);
</script>
<script xmlns="http://www.w3.org/1999/xhtml">
                    window.onload = function() {
                      if (false) {
                        JOT_setMobilePreview();
                      }
                      var loadTimer = window.jstiming.load;
                      loadTimer.tick("ol");
                      loadTimer["name"] = "load," + webspace.page.type + ",user_page";
                      window.jstiming.report(loadTimer, {}, 'https://gg.google.com/csi');
                    }
                  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
        JOT_insertAnalyticsCode(false,
            false);
      </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    var maestroRunner = new gsites.pages.view.SitesMaestroRunner(
        webspace, "en");
    maestroRunner.initListeners();
    maestroRunner.installEditRender();
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
  //<![CDATA[
    // Decorate any fastUI buttons on the page with a class of 'goog-button'.
    if (webspace.user.hasWriteAccess) {
      JOT_decorateButtons();
    }

    // Fires delayed events.
    (function() {
      JOT_fullyLoaded = true;
      var delayedEvents = JOT_delayedEvents;
      for (var x = 0; x < delayedEvents.length; x++) {
        var event = delayedEvents[x];
        JOT_postEvent(event.eventName, event.eventSrc, event.payload);
      }
      JOT_delayedEvents = null;
      JOT_postEvent('pageLoaded');
    })();
  //]]>
</script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    JOT_postEvent('decorateGvizCharts');
  </script>
<script type="text/javascript">
          JOT_setupPostRenderingManager();
        </script>
<script type="text/javascript">
          JOT_postEvent('renderPlus', null, 'sites-chrome-main');
        </script>
<script type="text/javascript">
          sites.codeembed.init();
        </script>
<div id="server-timer-div" style="display:none"> </div>
<script type="text/javascript">
          window.jstiming.load.tick('render');
          JOT_postEvent('usercontentrendered', this);
        </script>
</body>

<!-- Mirrored from sites.google.com/site/drrabhishekkumar/research-profile/research-experiences by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 25 Feb 2019 18:04:18 GMT -->
</html>
